easybuild/easyconfigs/b/Bazel/Bazel-3.7.2-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,45 @@
+name = 'Bazel'
+version = '3.7.2'
+
+homepage = 'https://bazel.io/'
+description = """Bazel is a build tool that builds code quickly and reliably.
+It is used to build the majority of Google's software."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = [
+    'https://github.com/bazelbuild/%(namelower)s/releases/download/%(version)s']
+sources = ['%(namelower)s-%(version)s-dist.zip']
+patches = [
+    '%(name)s-3.4.1-fix-grpc-protoc.patch',
+    'java15.patch',
+    '%(name)s-3.7.1_fix-protobuf-env.patch',
+    'gcc11-bazel-3.7.2.patch',
+    'gcc11-2-bazel-3.7.2.patch',
```

Comment:
@surak It would be nice to rename these patch files, to match the usual naming scheme we have (`<software name-version>_<short-descr>.patch`). Also, `gcc11-bazel-3.7.2.patch` and `gcc11-2-bazel-3.7.2.patch` can easily be combined into a single patch file.

I've made those changes myself in https://github.com/surak/easybuild-easyconfigs/pull/2

Reply:
Ok, agreed